stateoftheuniverse/widgets/constellations.py

```diff
@@ -0,0 +1,107 @@
+"""
+Get a list of constellations that will be visible from a location on the earth as a given time.
+"""
+#-------------------------
+#Imports
+#------------------------
+import warnings
+warnings.filterwarnings("ignore")
+from datetime import date as dt
+from astropy import units as u
+from astroplan import Observer
+from astropy.coordinates import SkyCoord, AltAz, get_constellation, EarthLocation
+from astropy.time import Time
+import numpy as np
+
+#-------------------------
+# Function Definitions
+#------------------------
+class ConstellationsWidget():
+	"""
+	
+	"""
+	def __init__(self, 
```

Comment:
The __init__ class needs to be changed around a bit to match the baseclass. Checkout prototype.py. It should just be:

```
    def __init__(self,
                longitude: Optional[float] = None,
                latitude: Optional[float] = None,
                datetime: Optional[datetime.datetime] = None):
        super().__init__(longitude=longitude,
                         latitude=latitude,
                         datetime=datetime)
```

instead of explicitly  giving it values. Let us know if you need any help!

Reply:
The other stuff you do in __init__ would be better in get_data :)